<!doctype html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<title>PhD Defense</title>

		<link rel="stylesheet" href="dist/reset.css">
		<link rel="stylesheet" href="dist/reveal.css">
		<link rel="stylesheet" href="theme.css">

		<!-- Theme used for syntax highlighted code -->
		<link rel="stylesheet" href="plugin/highlight/monokai.css">
	</head>
	<body>
		<div class="reveal">
			<div class="slides">


<section class="title-slide">
  <div class="title-logos">
	<img src="assets/logos/logos.svg"/>
  </div>

  <!-- You can wrap content in .title-stack to handle spacing -->
  <div class="title-stack">
    <h1 class="title-main">Restricted Boltzmann Machines as generic, generative, and interpretable models of the larval zebrafish brain</h1>
    <div class="title-author">Mattéo Dommanget‑Kott</div>

    <div class="title-supervisors">
      <div class="sec-heading">Supervisors</div>
      <ul class="sup-list">
        <li>Volker BORMUTH</li>
        <li>Georges DEBRÉGEAS</li>
      </ul>
    </div>

    <!-- <div class="title-jury-block"> -->
    <!--   <div class="sec-heading">Jury</div> -->
    <!-- 	<ul class="title-jury"> -->
    <!--   	<li> -->
    <!--     	<span class="person">Lilach AVITAN</span> -->
    <!--     	<span class="meta"><span class="role">Rapportrice</span> -->
    <!--   	</li> -->
    <!--   	<li> -->
    <!--     	<span class="person">Ruben PORTUGUES</span> -->
    <!--     	<span class="meta"><span class="role">Rapporteur</span> -->
    <!--   	</li> -->
    <!--   	<li> -->
    <!--     	<span class="person">N. Alex CAYCO GAJIC</span> -->
    <!--     	<span class="meta"><span class="role">Examinatrice</span> -->
    <!--   	</li> -->
    <!--   	<li> -->
    <!--     	<span class="person">Brice BATHELLIER</span> -->
    <!--     	<span class="meta"><span class="role">Examinateur</span> -->
    <!--   	</li> -->
    <!-- 	</ul> -->
    <!-- </div> -->

    <!-- <ul class="title-jury"> -->
    <!--   <li><span class="person">Lilach AVITAN</span> — <span class="role">Rapportrice</span>, <span class="aff">Institute Y</span></li> -->
    <!--   <li><span class="person">Ruben PORTUGUES</span> — <span class="role">Rapporteur</span>, <span class="aff">Lab Z</span></li> -->
    <!--   <li><span class="person">N. Alex CAYCO GAJIC</span> — <span class="role">Examinatrice</span>, <span class="aff">Institute Y</span></li> -->
    <!--   <li><span class="person">Brice BATHELLIER</span> — <span class="role">Examinateur</span>, <span class="aff">Lab Z</span></li> -->
    <!-- </ul> -->

    <!-- <div class="title-date">9 October 2025 — Paris</div> -->
  </div>
</section>



<section> <!-- INTRO SECTION -->

	<section>
		<!-- <h1> Introduction </h1> -->

		<div class=fragment>

			<div class="floating"
     			style="
						--floating-left: 12.1%; --floating-top: 13.0%; --floating-w: 360px; --floating-h: 351px;
     			">
				<img src="https://media1.giphy.com/media/v1.Y2lkPTZjMDliOTUyNTU4M3hnb3V1am9rMzZhbjQ3enRpeXFqcmd4aXZkaDBkdHh3YXV6ciZlcD12MV9naWZzX3NlYXJjaCZjdD1n/5ECm9620VUe0Uoq57j/200.gif" width="100%"/>
			</div>

			<div class="floating"
     			style="
						--floating-left: 47.7%; --floating-top: 11.5%; --floating-w: 376px; --floating-h: 343px;
     			">
				<img src="https://media2.giphy.com/media/v1.Y2lkPTZjMDliOTUyMGdieHNrMDM0MnlmOG1idmEwaWpvcjBzajhkMmY1aW4zcmU5dnNyMSZlcD12MV9naWZzX3NlYXJjaCZjdD1n/l4FGmO3MZkGng9Bp6/giphy.gif" width="100%"/>
			</div>

			<div class="floating"
     			style="
						--floating-left: 66.7%; --floating-top: 9.0%; --floating-w: 376px; --floating-h: 343px;
     			">
				<img src="https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExMTRleTRnN2t3Zmdqc3M2eXUzMHpuMTVhcm85ZnJyeGxtMW5pMWF0aiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/uIJBFZoOaifHf52MER/giphy.gif" width="100%"/>
			</div>

		</div>

		<div class="floating"
     		style="
					--floating-left: 11.9%; --floating-top: 8.9%; --floating-w: 1481px; --floating-h: 1017px;
     		">
			<img src="assets/intro/tripplettes.png" width="100%"/>
		</div>

		<aside class="notes">
			<p>During my PhD, I've been interested in understanding how to compare the brain activity of multiple individuals, and in particular, how to compare this activity when the people are not doing anything, and not thinking about anything in particular.</p>
		</aside>

	</section>



	<!-- <section> -->
	<!-- 	<h1> Introduction </h1> -->
	<!-- 	<h2> Goal </h2>  -->
	<!---->
	<!-- 	<div class="floating" -->
 <!--     		style=" -->
	<!-- 				--floating-left: -2.0%; --floating-top: 10.3%; --floating-w: 2000px; --floating-h: 107px; -->
 <!--     		"> -->
	<!-- 		<p>We want to evaluate how similar is the Spontaneous Brain Activity across individuals</p> -->
	<!-- 	</div> -->
	<!---->
	<!-- 	<div class="floating" -->
 <!--     		style=" -->
	<!-- 				--floating-left: -2.2%; --floating-top: 40.0%; --floating-w: 2004px; --floating-h: 221px; -->
 <!--     		"> -->
	<!-- 		<ol> -->
	<!-- 			<li>record Spontaneous Brain Activity → multiple individuals + whole-brain + neuron-scale</li> -->
	<!-- 			<li>find a shared representation of this activity across fish</li> -->
	<!-- 			<li>build of vocabulary of brain states</li> -->
	<!-- 			<li>compare the structure of the sequence of states</li> -->
	<!-- 		</ol> -->
	<!-- 	</div> -->
	<!---->
	<!-- </section> -->

	<section>
		<h1> Introduction </h1>

		<!-- <div class="floating fragment debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 2.0%; --floating-top: 23.0%; --floating-w: 348px; --floating-h: 286px; -->
  <!--    		"> -->
		<!-- 	<img src="https://www.mhs.ox.ac.uk/exhibits/revealing-the-brain/wp-content/uploads/fmriregions.gif" width="100%"/> -->
		<!-- </div> -->

		<div class="floating fragment"
     		style="
					--floating-left: 2.1%; --floating-top: 8.0%; --floating-w: 1782px; --floating-h: 954px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/neuro/fMRI_maps.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<!-- <div class="floating fragment debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 37.5%; --floating-top: 3.1%; --floating-w: 619px; --floating-h: 340px; -->
  <!--    		"> -->
  <!--    	<p>Morphological Registration</p> -->
		<!-- 	<img src="https://www.slicer.org/w/img_auth.php/thumb/c/cb/RegLib_C42_Xf2_registered.gif/300px-RegLib_C42_Xf2_registered.gif" width="100%"/> -->
		<!-- </div> -->

		<ul class="citations">
  		<!-- <li><a href="https://www.mhs.ox.ac.uk/exhibits/revealing-the-brain/regions/index.html" target="_blank" rel="noopener"> -->
    <!--   		<span class="author">MHS Oxford</span> <span class="journal">Exibition</span> <span class="year">2013</span> -->
    <!-- 	</a></li> -->
  		<!-- <li><a href="https://www.slicer.org/wiki/Documentation:Nightly:Registration:RegistrationLibrary:RegLib_C42" target="_blank" rel="noopener"> -->
    <!--   		<span class="author">3DSlicer</span> <span class="journal">Documentation</span> <span class="year">2013</span> -->
    <!-- 	</a></li> -->
  		<li><a href="https://europepmc.org/backend/ptpmcrender.fcgi?accid=PMC5576360&blobtype=pdf" target="_blank" rel="noopener">
      		<span class="author">Gordon & Laumann et al</span> <span class="journal">Neuron</span> <span class="year">2017</span>
    	</a></li>
		</ul>

		<aside class="notes">
			<p>Traditionally in neuroscience, when we want to learn something about how the brain works, we usually record the brain of participants while asking them to do something. So for example, in humans, you might record their brains using FMRI while they are watching a movie, moving their hands or feets, or whatever. </p>
			<p>Then you deduce what parts of the brain are related to, for example, moving their feet. But this only gives you information on that particular person. To learn something general about the human brain, we have to do the same experiment in many other participants and average their maps to get the brain regions common to everyone.</p>
			<p>However, this type of analysis doesn't work if you are interested in the brain activity which is not evoked by a particular behaviour or stimulus. </p>
		</aside>
  	<!-- <div class="entry">comparing brain activity has been done for a long time</div> -->
  	<!-- <div class="mainmessage"> recording=fRMI + stimuli or task → link activity with stim/task → maps identifying the functions of brain regions → compare maps across individuals using registration</div> -->
  	<!-- <div class="outlook">brains are different but share large scale functional organization</div> -->
	</section>


	<!-- <section> -->
	<!-- 	<h1> Introduction </h1> -->
 <!--  	<div class="entry">Even if we can find similar large brain structure that are specialized at doing different things, why do we expect brain activity to be the same. The brain controles the behavior, and it's easier to understand what an animal does than how an animal thinks.</div> -->
 <!--  	<div class="mainmessage">My favorite example : how flies clean themselfs. Behavior = stereotyped actions (vocabulary) + stereotyped structure of sequence</div> -->
 <!--  	<div class="outlook">you can find a shared representation of the behavior</div> -->
	<!-- </section> -->


	<section>
		<h1> Introduction </h1>
		<h2>Spontaneous Brain Activity</h2>

		<div class="floating fragment"
     		style="
					--floating-left: 0.0%; --floating-top: 13.8%; --floating-w: 1933px; --floating-h: 772px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/neuro/SA_def.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: -2.2%; --floating-top: 79.0%; --floating-w: 2007px; --floating-h: 120px;
     		">
			<p>Fundamental feature of Brain Function</p>
			<p>⠀</p>
			<p>linked with Brain Structure</p>
		</div>

		<aside class="notes">
			<p>This type of activity is called spontaneous brain activity, which is often described as ongoing and structured brain activity, which is unrelated to a current task or an explicit sensory input. Now I'll try and decompose this definition a little bit.</p>
			<p>“Ongoing” means that it is constantly present, when you are sleeping, when you are anaesthetised, when your mind is wandering. But, even when you are actively doing something. But the easiest way to observe it is at rest, when participants are asked to do nothing.</p>
			<p>The second part is structured. This is going to be a big part of this presentation, so I won't go into too much detail now, but the idea is that spontaneous brain activity is not randomness. It follows particular structures in the brain and it resembles normal brain activity.</p>
			<p>The last part means that this activity is not driven by a task or a stimulus. So when you average across trials or across individuals this activity will disappear. Because it’s an activity which isn’t reproducible. This is why studying SA, particularly across individuals, is such a challenge.</p>
			<p>Now, you might be wondering why we even bother with SA if it’s not related to doing stuff. It’s very important because it is involved in a whole bunch of different brain functions. For example, during development, both before and after you are born, SA plays a major role in shaping the structure of your brain. In recent years, some people have started to study how abnormal SA can be a marker for some psychiatric disorders like autism for example.</p>
			<p>And it's extremely involved in memory and learning as I'll show you in a bit, as well as in behaviour, which can be partially driven by SA.</p>
			<p>The fundamental idea is that spontaneous brain activity reflects the structure of your brain, and in particular how the different parts of your brains are connected with each other.</p>
		</aside>
	</section>

	<section>
		<h1> Introduction </h1>
		<h2>Spontaneous Activity reflects Brain Structure</h2>

		<div class="floating"
     		style="
					--floating-left: 1.5%; --floating-top: 14.6%; --floating-w: 1827px; --floating-h: 692px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/neuro/network.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: -1.8%; --floating-top: 77.2%; --floating-w: 1992px; --floating-h: 137px;
     		">
			<p>Spontaneous Activity = intrinsic revisiting of the network</p>
			<p>⠀</p>
			<p>⠀</p>
			<p>⇒ reflects network structure</p>
		</div>

		<!-- <div class="floating debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 29.4%; --floating-top: 13.5%; --floating-w: 517px; --floating-h: 434px; -->
  <!--    		"> -->
		<!-- 	<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5e/Place_Cell_Spiking_Activity_Example.png/330px-Place_Cell_Spiking_Activity_Example.png" width="100%"/> -->
		<!-- </div> -->
		<!---->
		<!-- <div class="floating debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 60.1%; --floating-top: 14.1%; --floating-w: 803px; --floating-h: 524px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/intro/neuro/replay.svg" width="100%"/> -->
		<!-- </div> -->

		<aside class="notes">
			<p>If you look at the structure of a brain, you'll see that there are brain regions which are more or less connected with each other. Some of them (in red) will be exciting each other, others will be inhibiting each other. And you have this very particular type of connectivity where not everyone is connected to everyone, but you have hubs and recurrent connections. If you look at the brain in more detail, at the much smaller scale of neurons, you're going to see that those circuits of neurons are also connected in this way with groups of neurons that are very strongly connected and then send connections to other groups and circuits that are further away. These small groups of neurons which are tightly connected, are called cell assemblies. And the idea is that throughout your experience, those neurons have learned to activate or deactivate altogether.</p>
			<p>Spontaneous activity happens like all brain activity in this structured network and therefore has to follow the rules of that network. As physicists we like to think about this in terms of energy. The brain can be in different states or configurations, which each correspond to a possible activity pattern of the whole brain. Some configurations would have higher energy, others low energy, and the brain evolves in this landscape by falling into the valleys or jumping over the hills. But you can imagine that some trajectories are easier to follow than others. </p>
			<p>The idea is that SA revisits this landscape without the biases or constraints that would be induced by a task or a stimulus. And therefore it’s helpful if we want to study how the brain is intrinsically organized.</p>
		</aside>
	</section>

	<section>
		<h1> Introduction </h1>
		<h2>Spontaneous Activity Revisits the Network</h2>

		<div class="floating"
     		style="
					--floating-left: 5.4%; --floating-top: 15.2%; --floating-w: 1665px; --floating-h: 763px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/neuro/replay.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<ul class="citations" style="bottom: 100px">
  		<li><a href="https://www.cell.com/neuron/fulltext/S0896-6273(10)00060-7" target="_blank" rel="noopener">
      		<span class="author">Gupta et al</span> <span class="journal">Neuron</span> <span class="year">2010</span>
    	</a></li>
		</ul>

		<aside class="notes">
			<p>An example of this I really like is an experiment in how rats move through a maze. So you have this maze where a rat has to follow those arrows so only experiences the maze in one direction.</p>
			<p>At the same time, it has a microscope attached to its head which records the activity of special neurons called place cells that only activate when the rat is at a particular location in the maze. So for example, the red place cell will only activate when the rat is at the top right corner of the maze. And so from the activity of those neurons, you can decode where the rat is according to its brain.</p>
			<p>During periods when that rat is immobile and not doing anything, SA revisits those place cells as if the rat was really moving around. Some trajectories look like what the rat has experienced, so in the direction of the arrows. Other trajectories are in reverse. But amazingly, some trajectories are shortcuts that were never experienced by the rat.</p>
			<p>So again, the idea is that SA is a way of revisiting brain networks, but it’s not just a simple replay of past experiences.</p>
		</aside>
	</section>

	<section>
		<h1> Introduction </h1>
		<h2>Stereotypy of Network Structure across Individuals</h2>

		<div class="floating"
     		style="
					--floating-left: 4.5%; --floating-top: 22.5%; --floating-w: 1047px; --floating-h: 478px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/neuro/network.svg" 
					data-base="Layer 1" 
					data-include="Layer 1"
				></div>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 42.5%; --floating-top: 15.7%; --floating-w: 966px; --floating-h: 600px;
     		">
			<img src="assets/intro/neuro/connectivity_stereo.svg" width="100%"/>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: -2.2%; --floating-top: 74.3%; --floating-w: 1992px; --floating-h: 137px;
     		">
			<p>Shared componants of brain structure ⇒ Shared componants of Spontaneous Activity</p>
		</div>

		<ul class="citations" style="bottom: 100px">
  		<li><a href="https://www.sciencedirect.com/science/article/pii/S0896627318302411" target="_blank" rel="noopener">
      		<span class="author">Gratton et al</span> <span class="journal">Neuron</span> <span class="year">2018</span>
    	</a></li>
		</ul>

		<aside class="notes">
			<p>So if we go back to our question of comparing the spontaneous brain activity of multiple individuals, maybe you see now that if the structure of the brain is the same for two individuals, then we expect them to have roughly the same spontaneous activity.</p>
			<p>This is something that has been tested in the past where people have looked at how the different parts of the brain are connected and have tried to deduce how much of this structure is shared amongst multiple individuals, how much is specific to each individual, and then a few other things around whether those networks change depending on the task.</p>
			<p>And what they have found is that most of the structure is either specific to an individual or shared by all the group, and those two components are about half and half. This means that a very large component of the brain structure is shared, and therefore we should have components of spontaneous activity which are also shared among individuals.</p>
		</aside>
	</section>

	<!-- <section> -->
	<!-- 	<h1> Introduction </h1> -->
 <!--  	<div class="entry">this is a very simple example but you can generalize it to much more complicated behavior.</div> -->
 <!--  	<div class="mainmessage">For example human movements. Infinite possibilities but in practice we can represent most behaviors with just a few joints (250 dof → 30 dof). We can simplify this even more : not every position equiprobable (forbiden + unlikely) + transitions</div> -->
 <!--  	<div class="outlook">if we can find a shared representation space for behavior, and behavior is controled by the brain → we should be able to find a simplified reprentation of brain activity which explains the behavior.</div> -->
	<!-- </section> -->

	<!-- <section> -->
	<!-- 	<h1> Introduction </h1> -->
	<!---->
	<!-- 	<div class="floating fragment" -->
 <!--     		style=" -->
	<!-- 				--floating-left: 6.2%; --floating-top: 15.4%; --floating-w: 1691px; --floating-h: 802px; -->
 <!--     		"> -->
	<!-- 		<img src="assets/intro/neuro/ICA.png" width="100%"/> -->
	<!-- 	</div> -->
	<!-- 	<div class="floating fragment" -->
 <!--     		style=" -->
	<!-- 				--floating-left: 6.15%; --floating-top: 17.15%; --floating-w: 1691px; --floating-h: 802px; -->
 <!--     		"> -->
	<!-- 			<video src="assets/intro/neuro/ICA.mp4"></video> -->
	<!-- 	</div> -->
	<!---->
	<!-- </section> -->

	<!-- <section> -->
	<!-- 	<h1> Introduction </h1> -->
	<!---->
	<!-- 	<div class="floating fragment debug" -->
 <!--     		style=" -->
	<!-- 				--floating-left: 0.1%; --floating-top: 24.9%; --floating-w: 1906px; --floating-h: 542px; -->
 <!--     		"> -->
	<!-- 			<div class="r-svg-layers"  -->
	<!-- 				data-src="assets/intro/neuro/fMRI_spont.svg"  -->
	<!-- 				data-base="Layer 1"  -->
	<!-- 			></div> -->
	<!-- 	</div> -->
	<!---->
	<!-- 	<ul class="citations"> -->
 <!--  		<li><a href="https://www.pnas.org/doi/10.1073/pnas.0601417103" target="_blank" rel="noopener"> -->
 <!--      		<span class="author">Damoiseaux et al.</span> <span class="journal">PNAS</span> <span class="year">2006</span> -->
 <!--    	</a></li> -->
	<!-- 	</ul> -->
	<!---->
	<!-- </section> -->

	<section>
		<h1> Outline </h1>

		<div class="floating"
     		style="
					--floating-left: -2.2%; --floating-top: 11.3%; --floating-w: 2007px; --floating-h: 128px;
     		">
			<p><b>Question :</b> Is Spontaneous Activity (SA) comparable across individuals?</p>
			<p>⠀</p>
			<p>⠀</p>
			<p>⇒ over the whole-brain at neuron-scale</p>
		</div>


		<div class="floating fragment"
     		style="
					--floating-left: -2.2%; --floating-top: 35.3%; --floating-w: 2022px; --floating-h: 709px;
     		">
			<ul style="line-height: 3.2;">
				<li>Recording whole-brain single-neuron SA</li>
				<li>Statistical model → representation of SA</li>
				<li>Align representation across individuals</li>
				<li>Compare SA dynamics across individuals</li>
			</ul>
		</div>

		<aside class="notes">
			<p>Our goal is going to be to compare SA across individuals. However, as I told you, the brain is structured at multiple scales, from the whole-brain to the neurons. And in humans we cannot record the activity of neurons.</p>
			<p>So throughout the rest of this presentation I’m going to explain how we record the whole-brain activity of an animal at the scale of neurons.</p>
			<p>Then I'll show you a statistical model that we use to find the components of SA from those neurons.</p>
			<p>Then I'll present a new method that we have developed to align this statistical model across multiple individuals.</p>
			<p>And finally, we’ll use it to compare the dynamics of the SA across those individuals.</p>
		</aside>
	</section>

	<!-- <section> -->
	<!-- 	<h1> Introduction </h1> -->
	<!---->
	<!-- 	<div class="floating debug" -->
 <!--     		style=" -->
	<!-- 				--floating-left: -2.2%; --floating-top: 15.9%; --floating-w: 581px; --floating-h: 312px; -->
 <!--     		"> -->
	<!-- 			<iframe width="560" height="315" src="https://www.youtube.com/embed/GgZHziFWR7M?si=B8FKE14Gp0ljgnNK" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe> -->
	<!-- 	</div> -->
	<!---->
	<!-- 	<div class="floating debug" -->
 <!--     		style=" -->
	<!-- 				--floating-left: -2.2%; --floating-top: 64.3%; --floating-w: 887px; --floating-h: 307px; -->
 <!--     		"> -->
	<!-- 		<img src="assets/intro/neuro/elegans_manifold.svg" width="100%"/> -->
	<!-- 	</div> -->
	<!---->
	<!-- 	<div class="floating debug" -->
 <!--     		style=" -->
	<!-- 				--floating-left: 32.7%; --floating-top: 15.9%; --floating-w: 900px; --floating-h: 249px; -->
 <!--     		"> -->
	<!-- 		<img src="https://www.sfu.ca/biology/faculty/hutter/hutterlab/research/pictures/Ce_ns_label.gif" width="100%"/> -->
	<!-- 	</div> -->
	<!---->
	<!-- 	<ul class="citations"> -->
 <!--  		<li><a href="https://elifesciences.org/articles/46814" target="_blank" rel="noopener"> -->
 <!--      		<span class="author">Brennan & Proekt</span> <span class="journal">eLife</span> <span class="year">2019</span> -->
 <!--    	</a></li> -->
	<!-- 	</ul> -->
	<!---->
 <!--  	<div class="entry">doing whole brain analysis in small animals where neurons can be identified</div> -->
 <!--  	<div class="mainmessage">C. elegens : same neurons can be identified in multiple individuals. maps to latent space representation of behavior.</div> -->
 <!--  	<div class="outlook">Very simple brain → we would like to study vertebrates</div> -->
	<!-- </section> -->

	<!-- <section> -->
	<!-- 	<h1> Introduction </h1> -->
	<!-- 	<h2>The gap</h2> -->
 <!--  	<div class="entry">Classical method for comparing brains</div> -->
 <!--  	<div class="mainmessage">How to compare Spontaneous brain activity at single-cell resolution in vertebrates ? + no neuron-neuron correspondances + no behavior or stimuli for alignment</div> -->
 <!--  	<div class="outlook">next I'll present the animal model we used, I'll present the methods I developed to solve this methodological problem, and then I'll try to use it to brain dynamics across individuals</div> -->
	<!-- </section> -->

	<section>
		<h1> Recording Whole-Brain Single-neuron Activity </h1>
		<h2>Zebrafish</h2>

		<div class="floating"
     		style="
					--floating-left: 1.2%; --floating-top: 12.1%; --floating-w: 390px; --floating-h: 313px;
     		">
     	Adult
			<img src="assets/intro/zebra/adult.png" width="100%"/>
		</div>

		<div class="floating fragment" data-fragment-index="1"
     		style="
					--floating-left: 1.3%; --floating-top: 46.1%; --floating-w: 389px; --floating-h: 548px;
     		">
     	Larva
			<img src="assets/intro/zebra/rice.png" width="100%"/>
		</div>

		<div class="floating fragment" data-fragment-index="2"
     		style="
					--floating-left: 24.7%; --floating-top: 16.5%; --floating-w: 661px; --floating-h: 880px;
     		">
			<video loop="true" src="assets/intro/zebra/lots_of_little_fish.mp4"></video>
		</div>

		<div class="floating fragment" data-fragment-index="3"
     		style="
					--floating-left: 62.7%; --floating-top: 10.7%; --floating-w: 592px; --floating-h: 649px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/zebra/fluo_annotated.svg" 
					data-base="Layer 1" 
				></div>
			<!-- <img src="assets/intro/zebra/fluo_annotated.svg" width="100%"/> -->
		</div>

		<div class="floating"
     		style="
					--floating-left: 63.6%; --floating-top: 66.6%; --floating-w: 559px; --floating-h: 254px;
     		">
			<ul>
				<li class="fragment" data-fragment-index="2">Rich behavior</li>
				<li class="fragment" data-fragment-index="3">Transparent</li>
				<li class="fragment" data-fragment-index="4">Small brain (~100k neurons)</li>
				<li class="fragment" data-fragment-index="5">Genetic tools (GCaMP, ...)</li>
			</ul>
		</div>

		<!-- <div class="floating debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 79.5%; --floating-top: 11.9%; --floating-w: 348px; --floating-h: 286px; -->
  <!--    		"> -->
		<!-- 	<video loop="true" src="assets/intro/zebra/gcamp_slide_show.mp4"></video> -->
		<!-- </div> -->


		<!-- <ul class="citations" style="bottom: 100px"> -->
  <!-- 		<li><a href="https://www.biorxiv.org/content/10.1101/2024.11.22.624881v1" target="_blank" rel="noopener"> -->
  <!--     		<span class="author">Dommanget-Kott & Fernandez-de-Cossio-Diaz, et al.</span> <span class="journal">bioRxiv</span> <span class="year">2024</span> -->
  <!--   	</a></li> -->
		<!-- </ul> -->


  	<!-- <div class="entry">??</div> -->
  	<!-- <div class="mainmessage">Zebra larvae are perfect for vertebrate whole brain recordings (transparent, ...) + behavior (mention HMM paper)</div> -->
  	<!-- <div class="outlook">??</div> -->

		<aside class="notes">
			<p>Instead of humans we use zebrafish which look like this when they are adults, but actually we use them when they are larvae. So when they're around five days old, at that age, they're very small as you can see compared to the grain of rice. But they already have quite a rich behaviour where they can already move around, they can hunt for food or escape from predators.</p>
			<p>The reason they are so useful to us is because they are almost entirely transparent. So you can see their organs including their brains, which are relatively small and contain about a hundred thousand neurons. But they're vertebrates like us, so some results that we obtain in zebrafish can be then extrapolated to humans or other vertebrates.</p>
			<p>We can modify them genetically so that each neuron expresses a fluorescent protein, which lets you look at their brain. So here everything in green is the brain.</p>
		</aside>
	</section>

	<section>
		<h1> Recording Whole-Brain Single-neuron Activity </h1>
		<h2>Light Sheet Microscopy</h2>

		<div class="floating fragment"
     		style="
					--floating-left: 2.1%; --floating-top: 12.8%; --floating-w: 861px; --floating-h: 925px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/zebra/fluo_diag.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<div class="floating fragment"
     		style="
				--floating-left: 48.4%; --floating-top: 5.0%; --floating-w: 517px; --floating-h: 1046px;
     		">
			<video loop="true" src="assets/intro/zebra/LSM.mp4"></video>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 74.6%; --floating-top: 37.5%; --floating-w: 459px; --floating-h: 265px;
     		">
     	<ul>
			<li>3D recording @2.5Hz</li>
			<li>~45,000 neurons</li>
			<li>25mins</li>
			<li>paralysed</li>
     	</ul>
		</div>

		<aside class="notes">
			<p>The way this works is that each neuron produces a whole bunch of proteins called GCaMPs. When the neuron activates calcium enters the cell and binds to the protein, changing its shape, which makes it fluorescent. And so now, if we illuminate the neuron with blue light, it will glow in green. This allows us to follow the activity of the neuron by following over time how fluorescent it is. From this we can deduce the spike rate of the neuron, and in our case we only look at whether the neuron is active or not at each time point.</p>
			<p>To record the whole brain, we quickly scan a laser beam across the brain, which allows us to record the activity of all the neurons in that layer. Then we can move the laser beam up and down to scan the whole brain, in 3D at 2.5 volumes per second. This allows us to record the activity of about 45k neurons. All the results I’ll show you today are from 25mins recordings, in paralysed fish.</p>
		</aside>
	</section>

	<section>
		<h1> Recording Whole-Brain Single-neuron Activity </h1>
		<h2>Light Sheet Microscopy</h2>
  	<div class="entry"></div>

		<div class="floating"
     		style="
				--floating-left: -0.2%; --floating-top: 18.6%; --floating-w: 1931px; --floating-h: 741px;
     		">
			<video loop="true" src="assets/multiRBM/Movie1.mp4"></video>
		</div>

		<aside class="notes">
			<p>Here you see the activity of 6 fish, from a top view where each dot is green if the neuron is active. This is SA, so one thing you can see straight away is that their activity is quite different. But you might be able to convince yourself that amongst all that randomness, there is some structure. For example, groups of neurons tend to all activate together. And also you can maybe convince yourself that some of this structure is shared by all the fish.</p>
		</aside>
	</section>


</section> <!-- INTRO SECTION -->


<section> <!-- DEAMON -->
	<section>
		<!-- <div class="floating debug" -->
  <!--    		style=" -->
		<!-- 		--floating-left: -0.2%; --floating-top: 18.6%; --floating-w: 1931px; --floating-h: 741px; -->
  <!--    		"> -->
		<!-- 	<video loop="true" src="assets/intro/deamon/fish_and_tree.mp4"></video> -->
		<!-- </div> -->
			<video loop="true" src="assets/intro/deamon/fish_and_tree.mp4" style="position:absolute; top:7%; left:1%;" width="100%"></video>
		<aside class="notes">
			<p>Now, I don't know about you, but this reminds me a lot of a Christmas tree. And would like to understand what is the structure behind this chaos.</p>
		</aside>
	</section>

	<section>
			<video src="assets/intro/deamon/deamon1_nl.mp4" style="position:absolute; top:7%; left:1%;" width="100%"></video>
		<aside class="notes">
			<p>So we are going to imagine that the lights are controlled by a demon. Let’s call it Hinton’s demon.</p>
		</aside>
	</section>

	<section>
			<video loop="true" src="assets/intro/deamon/deamon2_l.mp4" style="position:absolute; top:7%; left:1%;" width="100%"></video>
		<aside class="notes">
			<p>It controls the lights by connecting their wires to a battery. However it only has 6 arms and so it cannot control all of those lights.</p>
		</aside>
	</section>

	<section>
			<video src="assets/intro/deamon/deamon3_nl.mp4" style="position:absolute; top:7%; left:1%;" width="100%"></video>
		<aside class="notes">
			<p>So the demon groups the wires in bundles, ... </p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
		</aside>
	</section>

	<section>
			<video loop="true" src="assets/intro/deamon/deamon4_l.mp4" style="position:absolute; top:7%; left:1%;" width="100%"></video>
		<aside class="notes">
			<p>... which each connect to a group of lights. </p>
		</aside>
	</section>

	<section>
			<video src="assets/intro/deamon/deamon5_nl.mp4" style="position:absolute; top:7%; left:1%;" width="100%"></video>
		<aside class="notes">
			<p>However, the demon is devious, and it wants to trick us, so it ruffles the ends of the wires to make them random.</p>
		</aside>
	</section>

	<section>
			<video loop="true" src="assets/intro/deamon/deamon6_l.mp4" style="position:absolute; top:7%; left:1%;" width="100%"></video>
		<aside class="notes">
			<p>No when it connects the bundles to the battery, some wires touch and others don’t, which adds some randomness to the lights.</p>
		</aside>
	</section>

	<section>
			<video src="assets/intro/deamon/deamon7.mp4" style="position:absolute; top:7%; left:1%;" width="100%"></video>
		<aside class="notes">
			<p>And the demon can go all in! But now that you understand what the demon is doing, maybe you understand the lights are not chaotic at all.</p>
		</aside>
	</section>

	<section>
			<img src="assets/intro/deamon/deamon_alone2.png" style="position:absolute; top:5.3%; left:1%;" width="100%"/>

		<div class="floating"
     		style="
					--floating-left: -2.2%; --floating-top: 30.8%; --floating-w: 781px; --floating-h: 685px;
     		">
     	<p>Simple Rules with added Noise</p>
			<p>⠀</p>
			<p>⠀</p>
			<p>⠀</p>
			<p>⠀</p>
			<p>⠀</p>
			<p>⠀</p>
     	<p>Representation of the collective</p>
			<p>⠀</p>
			<p>activity patterns of lights</p>
		</div>
		<aside class="notes">
			<p>It’s just simple rules with added noise, yet it provides a good representation of how the christmas tree lights are activating.</p>
		</aside>
	</section>

	<section>
			<img src="assets/intro/deamon/deamon_alone2.png" style="position:absolute; top:5.3%; left:1%;" width="100%"/>

		<div class="floating"
     		style="
					--floating-left: 0.6%; --floating-top: -0.0%; --floating-w: 701px; --floating-h: 1101px;
     		">
			<img src="assets/intro/zebra/all_records.png" width="100%"/>
		</div>

		<aside class="notes">
			<p>A large part of my PhD work has been to see if it is possible to explain the brain activity of multiple zebrafish using the same demon.</p>
			<p>Now in reality, I haven't been searching for a real demon for the last 4 years.</p>
		</aside>
	</section>

</section> <!-- DEAMON -->



<section> <!-- RBMs SECTION -->

	<!-- <section> -->
	<!-- 	<div class="floating" -->
 <!--     		style=" -->
	<!-- 				--floating-left: -0.0%; --floating-top: 2.0%; --floating-w: 1919px; --floating-h: 1099px; -->
 <!--     		"> -->
	<!-- 		<video loop="true" src="assets/intro/0001-1000.mp4"></video> -->
	<!-- 	</div> -->
	<!-- </section> -->
	<!---->
	<!-- <section> -->
	<!-- 	<div class="floating" -->
 <!--     		style=" -->
	<!-- 				--floating-left: 26.4%; --floating-top: 24.8%; --floating-w: 862px; --floating-h: 707px; -->
 <!--     		"> -->
	<!-- 		<img src="assets/intro/deamon_multi.svg" width="100%"/> -->
	<!-- 	</div> -->
	<!-- </section> -->

	<section>
		<h1> Restricted Boltzmann Machines </h1>
		<h2> A Probabilistic Model </h2>

		<div class="floating"
     		style="
					--floating-left: 85.4%; --floating-top: -0.8%; --floating-w: 248px; --floating-h: 340px;
     		">
				<p style="font-size: 0.8em;">Geoffrey Hinton</p>
			<img src="https://chartwellspeakers.b-cdn.net/wp-content/uploads/2023/09/geoffhinton_14977197-e1697029454322.webp" width="100%"/>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: -2.3%; --floating-top: 0.3%; --floating-w: 2005px; --floating-h: 1074px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/RBM/math.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 22.6%; --floating-top: 8.5%; --floating-w: 936px; --floating-h: 405px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/intro/RBM/diag_simple.svg" width="100%"/> -->
		<!-- </div> -->


		<!-- <div class="floating debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -0.3%; --floating-top: 42.8%; --floating-w: 1005px; --floating-h: 205px; -->
  <!--    		"> -->
		<!-- 	<div style="text-align: left;"> -->
		<!-- 		<p>visible configuration : $\mathbf v=(v_1,\dots,v_N)\in\{0,1\}^N$</p> -->
		<!-- 		<p>hidden configuration : $\mathbf h=(h_1,\dots,h_M)\in\mathbb{R}^M$</p> -->
		<!-- 	</div> -->
		<!-- </div> -->
		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 15.2%; --floating-top: 43.8%; --floating-w: 611px; --floating-h: 152px; -->
  <!--    		"> -->
		<!-- 		<p>visible configuration</p> -->
				<!-- $$\mathbf v=(v_1,\dots,v_N)\in\{\text{on},\text{off}\}^N$$ -->
				<!-- <p>hidden configuration : $\mathbf h=(h_1,\dots,h_M)\in\mathbb{R}^M$</p> -->
		<!-- </div> -->
		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 53.4%; --floating-top: 43.7%; --floating-w: 562px; --floating-h: 151px; -->
  <!--    		"> -->
		<!-- 		<p>hidden configuration</p> -->
				<!-- $$\mathbf h=(h_1,\dots,h_M)\in\mathbb{R}^M$$ -->
		<!-- </div> -->

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 6.5%; --floating-top: 60.1%; --floating-w: 1083px; --floating-h: 375px; -->
  <!--    		"> -->
		<!-- 	$$ -->
		<!-- 		\begin{aligned} -->
  <!--         P(\mathbf v,\mathbf h) &= \frac{1}{Z}\, -->
  <!--         e^{-E(\mathbf v,\mathbf h)}\\ -->
  <!--         E(\mathbf v,\mathbf h) &= -->
  <!--             -\sum_{i=1}^{N}g_i v_i -->
  <!--             +\sum_{\mu=1}^{M}\mathcal{U}_\mu(h_\mu) -->
  <!--             -\sum_{i,\mu} w_{i\mu} v_i h_\mu -->
  <!--         \end{aligned} -->
		<!-- 	$$ -->
		<!-- </div> -->

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 65.2%; --floating-top: 64.5%; --floating-w: 432px; --floating-h: 268px; -->
  <!--    		"> -->
		<!-- 	parameters $ -->
		<!-- 	\begin{cases} -->
		<!-- 	&g_i\\ -->
		<!-- 	&\mathcal{U}_\mu\\ -->
		<!-- 	&w_{i\mu} -->
		<!-- 	\end{cases} -->
		<!-- 	$ -->
		<!-- </div> -->

		<!-- <div class="floating fragment debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 56.2%; --floating-top: 48.2%; --floating-w: 380px; --floating-h: 388px; -->
  <!--    		"> -->
  <!--    		<p>Continuous hiddens</p> -->
		<!-- 		<div class="r-svg-layers"  -->
		<!-- 			data-src="assets/multiRBM/TS_Ph.svg"  -->
		<!-- 			data-base="Layer 1"  -->
		<!-- 			data-include="Layer 1" -->
		<!-- 		></div> -->
		<!-- </div> -->

		<!-- <div class="floating fragment debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 77.0%; --floating-top: 48.2%; --floating-w: 380px; --floating-h: 388px; -->
  <!--    		"> -->
  <!--    		<p>Continuous weights</p> -->
				<!-- <div class="r-svg-layers"  -->
				<!-- 	data-src="assets/multiRBM/TS_Ph.svg"  -->
				<!-- 	data-base="Layer 1"  -->
				<!-- 	data-include="Layer 1" -->
				<!-- ></div> -->
		<!-- </div> -->

  	<!-- <div class="entry">Geofrey Hinton</div> -->
  	<!-- <div class="mainmessage">Diagram and math + (v to multiple h, continuous h and distrib, ...) + Gibbs</div> -->
  	<!-- <div class="outlook">application to zebrafish by Thijs and Jerome</div> -->

		<aside class="notes">
			<p>Instead I’ve been using a mathematical model of this demon, which is called the Restricted Boltzmann Machine which has been largely developed by Geoffrey Hinton.</p>
			<p>The idea behind this model is to have two layers, one which represents the neurons on the left and green, where each neuron can be either on or off, and one on the right that represents hidden units, which are equivalent to the arms of the demon, and will provide a representation of the brain activity.</p>
			<p>Mathematically, the model is described by a joint probability distribution over the neuron and hidden unit configurations. It's a boltman distribution defined from an energy function with 3 terms.</p>
			<p>The first term describes the tendency of each neuron to be ON or OFF.</p>
			<p>The second term describes how each hidden unit can be more or less active, but with continuous values.</p>
			<p>And the last term describes the interactions between the neurons and the hidden units, mediated by weights that connect directly the neurons and the hidden units. So each weight in a continuous value, which can be positive of negative</p>
			<p>So the entire model is parametrized by those 3 parameters g and U which I won’t discuss much here, and the weights w_i_mu which we’ll discuss in length throughout this presentation.</p>
		</aside>
	</section>

	<section>
		<h1> Restricted Boltzmann Machines </h1>
		<h2> A Generative Model </h2>

		<div class="floating"
     		style="
					--floating-left: -2.1%; --floating-top: 11.1%; --floating-w: 936px; --floating-h: 405px;
     		">
			<img src="assets/intro/RBM/diag_simple.svg" width="100%"/>
		</div>


		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -2.0%; --floating-top: 52.0%; --floating-w: 2001px; --floating-h: 82px; -->
  <!--    		"> -->
		<!-- 	same statistics ⇒ $\langle\mathbf{v}\rangle$, $\langle\mathbf{v}\mathbf{v}\rangle$, $\langle\mathbf{v}\mathbf{h}\rangle$, $\langle\mathbf{h}\rangle$, $\langle\mathbf{h}\mathbf{h}\rangle$ -->
		<!-- </div> -->

		<div class="floating fragment"
     		style="
					--floating-left: 56.4%; --floating-top: 11.7%; --floating-w: 619px; --floating-h: 95px;
     		">
			maximize $\mathcal{L} = \langle \log P(\mathbf{v}) \rangle_\text{data}$
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 58.5%; --floating-top: 22.8%; --floating-w: 524px; --floating-h: 248px;
     		">
			learn parameters $
			\begin{cases}
			&g_i\\
			&\mathcal{U}_\mu\\
			&w_{i\mu}
			\end{cases}
			$
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 25.2%; --floating-top: 47.3%; --floating-w: 919px; --floating-h: 529px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/RBM/gibbs2.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<aside class="notes">
			<p>The goal of this model is to capture the distribution of neuronal configurations P(v), and so the model is trained by modifying the parameters to maximize the likelihood of the data. However, it’s impossible to compute P(v) directly.</p>
			<p>But we are saved by the fact that this model is generative. The idea is that the joint distribution P(v,h) can be decomposed into 2 conditional distributions which allow us to move between the visible and hidden representations of the data. So for example, we can start from this brain configuration, where some neurons are active and some neurons are inactive, and then use P(h|v) to generate a hidden representation. We can also do the same the other way around, using P(v|h) to generate a visible configuration from this hidden representation.</p>
			<p>By repeating this procedure over and over again we can generate a whole bunch of new configurations which should look like the genuine recorded brain activity, meaning that they follow P(v).</p>
		</aside>
	</section>

	<section>
		<h1> Restricted Boltzmann Machines </h1>
		<h2> as models of the zebrafish brain </h2>

		<div class="floating"
     		style="
					--floating-left: -1.3%; --floating-top: 63.8%; --floating-w: 240px; --floating-h: 307px;
     		">
			<p style="font-size: 0.7em;">Thijs van der Plas</p>
			<img src="https://research.wur.nl/files-asset/126634744/GetMedia.ashx_id_123483_ts_1743673080?w=160&f=jpg" width="100%"/>
		</div>
		<div class="floating"
     		style="
					--floating-left: 9.9%; --floating-top: 63.8%; --floating-w: 240px; --floating-h: 314px;
     		">
			<p style="font-size: 0.7em;">Jérôme Tubiana</p>
			<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRd5M6DoNByKuL3ONCGAu2a3WH3Wke_DdHgPQ&s" width="100%"/>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 48.4%; --floating-top: 26.2%; --floating-w: 960px; --floating-h: 493px;
     		">
			<img src="assets/intro/RBM/stats2.svg" width="100%"/>
		</div>


		<div class="floating"
     		style="
					--floating-left: -2.5%; --floating-top: 22.1%; --floating-w: 1276px; --floating-h: 484px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/RBM/diag.svg" 
					data-base="Layer 1" 
					data-include="Layer 1"
				></div>
		</div>

		<!---->
		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 45.5%; --floating-top: 75.9%; --floating-w: 799px; --floating-h: 109px; -->
  <!--    		"> -->
		<!-- 	<p>Representation of Neuronal Configuration</p> -->
		<!-- 	<p>⠀</p> -->
		<!-- 	<p>⇒ Hidden Units ≈ Cell Assemblies</p> -->
		<!-- </div> -->


		<ul class="citations" style="bottom: 100px">
  		<li><a href="https://elifesciences.org/articles/83139" target="_blank" rel="noopener">
      		<span class="author">van der Plas & Tubiana et al.</span> <span class="journal">eLife</span> <span class="year">2023</span>
    	</a></li>
		</ul>

  	<!-- <div class="entry"></div> -->
  	<!-- <div class="mainmessage">HU maps + filtering + (cell assemblies)</div> -->
  	<!-- <div class="outlook"></div> -->

		<aside class="notes">
			<p>This model has already been used in the past by Thijs van der Plas, Jerome Tubiana and others in the lab to study the whole-brain spontaneous activity of individual zebrafish larvae.</p>
			<p>They've shown that you can train an RBM from this type of dataset. For example, the model is able to reproduce the mean activity of the neurons. So on the X axis you have the mean activity of each neuron as observed in the data and on the Y axis you have the mean activity of neurons from model-generated configurations. And as you can see, the two fit perfectly along the identity. It's the same for the pairwise correlations between the neurons.</p>
			<p>So this model is able to generate neuronal configurations which look statistically like real zebrafish brain activity.</p>
		</aside>
	</section>

	<section>
		<h1> Restricted Boltzmann Machines </h1>
		<h2> Representing Brain Activity as a Composition of Cell Assemblies </h2>

		<div class="floating"
     		style="
					--floating-left: -1.3%; --floating-top: 63.8%; --floating-w: 240px; --floating-h: 307px;
     		">
			<p style="font-size: 0.7em;">Thijs van der Plas</p>
			<img src="https://research.wur.nl/files-asset/126634744/GetMedia.ashx_id_123483_ts_1743673080?w=160&f=jpg" width="100%"/>
		</div>
		<div class="floating"
     		style="
					--floating-left: 9.9%; --floating-top: 63.8%; --floating-w: 240px; --floating-h: 314px;
     		">
			<p style="font-size: 0.7em;">Jérôme Tubiana</p>
			<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRd5M6DoNByKuL3ONCGAu2a3WH3Wke_DdHgPQ&s" width="100%"/>
		</div>

		<div class="floating"
     		style="
					--floating-left: 5.5%; --floating-top: 17.6%; --floating-w: 1276px; --floating-h: 484px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/RBM/diag_complex.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 59.4%; --floating-top: 3.5%; --floating-w: 697px; --floating-h: 1040px;
     		">
			<img src="assets/intro/RBM/elife.svg" width="100%"/>
		</div>

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 21.3%; --floating-top: 81.2%; --floating-w: 1540px; --floating-h: 70px; -->
  <!--    		"> -->
		<!-- 	  <p>cell assembly = group of co-activating neurons, usually spatially localized</p> -->
		<!-- </div> -->

		<ul class="citations" style="bottom: 100px">
  		<li><a href="https://elifesciences.org/articles/83139" target="_blank" rel="noopener">
      		<span class="author">van der Plas & Tubiana et al.</span> <span class="journal">eLife</span> <span class="year">2023</span>
    	</a></li>
		</ul>

		<aside class="notes">
			<p>Amongst many other things, they have shown that the RBM provides a representation of brain activity as a composition of cell assemblies. I'm going to unpack this statement because it's an extremely important one for this presentation.</p>
			<p>The idea is that RBMs capture the co-activation motifs of neurons by grouping them to the same hidden units. So if we imagine that those five neurons have a tendency of always being active together, then they will all be grouped to this blue hidden unit.</p>
			<p>And it's the same for other groups of neurons that also tend to co-activate, and each of them will be linked to a hidden unit.</p>
			<p>And so this creates groups of neurons, throughout the brains, that are each linked to a different hidden unit.</p>
			<p>An important result from Thijs and Jerome is that these groups describe populations of neurons which are known and look like known functional circuits. For example, this one here looks a lot like pre-motor neurons that control the movement of the tail.</p>
			<p>An important feature of RBMs is that one neuron can be connected to more than 1 hidden unit, like this one for example.</p>
		</aside>
	</section>

	<section>
		<h1> Restricted Boltzmann Machines </h1>
		<h2> Representing Brain Activity as a Composition of Cell Assemblies </h2>


		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -1.3%; --floating-top: 63.8%; --floating-w: 240px; --floating-h: 307px; -->
  <!--    		"> -->
		<!-- 	<p style="font-size: 0.7em;">Thijs van der Plas</p> -->
		<!-- 	<img src="https://research.wur.nl/files-asset/126634744/GetMedia.ashx_id_123483_ts_1743673080?w=160&f=jpg" width="100%"/> -->
		<!-- </div> -->
		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 9.9%; --floating-top: 63.8%; --floating-w: 240px; --floating-h: 314px; -->
  <!--    		"> -->
		<!-- 	<p style="font-size: 0.7em;">Jérôme Tubiana</p> -->
		<!-- 	<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRd5M6DoNByKuL3ONCGAu2a3WH3Wke_DdHgPQ&s" width="100%"/> -->
		<!-- </div> -->

		<div class="floating fragment"
     		style="
					--floating-left: 8.7%; --floating-top: 15.8%; --floating-w: 1519px; --floating-h: 583px;
     		">
			<img src="assets/intro/RBM/compositionality.jpg" width="100%"/>
		</div>
		<div class="floating fragment"
     		style="
					--floating-left: 8.7%; --floating-top: 17.45%; --floating-w: 1519px; --floating-h: 583px;
     		">
			<video loop="true" src="assets/intro/RBM/compositionality.mp4" width="100%"></video>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: -2.3%; --floating-top: 73.6%; --floating-w: 2004px; --floating-h: 167px;
     		">
			<p>hidden units ≠ brain states</p>
			<p>⠀</p>
			<p>⠀</p>
			<p>⠀</p>
			<p>brain states = superposition of a <b>few</b> hidden units</p>
		</div>

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 21.3%; --floating-top: 81.2%; --floating-w: 1540px; --floating-h: 70px; -->
  <!--    		"> -->
		<!-- 	  <p>cell assembly = group of co-activating neurons, usually spatially localized</p> -->
		<!-- </div> -->

		<!-- <ul class="citations" style="bottom: 100px"> -->
  <!-- 		<li><a href="https://elifesciences.org/articles/83139" target="_blank" rel="noopener"> -->
  <!--     		<span class="author">van der Plas & Tubiana et al.</span> <span class="journal">eLife</span> <span class="year">2023</span> -->
  <!--   	</a></li> -->
		<!-- </ul> -->

		<aside class="notes">
			<p>This overlap combined with the fact that weights can have positive and negative values means that we can compose a large variety of states.</p>
			<p>Here I'm showing you a toy example with 3 hidden units. Each dot is a neuron and the color shows how they connect to each hidden unit, with blue for negative weights, red for positive weights, and white when they're not connected.</p>
			<p>Now by combining those different hidden units with different amounts, which is what those h mean, we can create a large variety of different activity patterns for this neuronal population.</p>
			<p>This is a very strong property called compositionality, which means that the complex activity patterns at the scale of the whole brain, can be explained as a composition of a small number of cell assemblies.</p>
			<p>So hidden units don’t represent brain states, a brain state is a composition of a few hidden units.</p>
		</aside>
	</section>

	<section>
		<h1> Restricted Boltzmann Machines </h1>
		<h2> Provide Degenerate Representations </h2>

		<div class="floating"
     		style="
					--floating-left: 3.7%; --floating-top: 12.5%; --floating-w: 1730px; --floating-h: 96px;
     		">
			<p><b>Goal :</b> Find a Common Representation for the Spontaneous Brain Activity of multiple fish.</p>
		</div>


		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -2.0%; --floating-top: 22.0%; --floating-w: 1033px; --floating-h: 212px; -->
  <!--    		"> -->
		<!-- 	<ul> -->
		<!-- 		<li class="fragment">Hidden Units ≈ known Functional Networks</li> -->
		<!-- 		<li class="fragment">Functional Networks are shared by all individuals</li> -->
		<!-- 	</ul> -->
		<!-- 	<p>⠀</p> -->
		<!-- 	<p class="fragment">⇒ multiple RBMs shoud provide the same representation</p> -->
		<!-- </div> -->
		<!---->
		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 57.7%; --floating-top: 27.0%; --floating-w: 758px; --floating-h: 69px; -->
  <!--    		"> -->
		<!-- 	<p class="fragment">RBMs are stochastic ⇒ variable solutions</p> -->
		<!-- </div> -->

		<div class="floating fragment"
     		style="
					--floating-left: 0.5%; --floating-top: 31.8%; --floating-w: 777px; --floating-h: 547px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/RBM/singlefish_multiRBMs.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<div class="floating fragment"
     		style="
				--floating-left: 41.4%; --floating-top: 35.9%; --floating-w: 1090px; --floating-h: 446px;
     		">
			<img src="assets/multiRBM/degen_weights.svg" width="100%"/>
		</div>

		<aside class="notes">
			<p>Now, our goal is to find a representation of brain activity which is shared amongst many fish, not just 1.</p>
			<p>But it seems that RBMs are quite good candidates for doing this as hidden units capture functional networks which we know are shared across fish.</p>
			<p>So the idea would be to train 1 RBM per fish, with the hope that their respective hidden units describe the same neuronal populations.</p>
			<p>However, even if those 2 fish were identical, there is a risk that the 2 RBMs would converge to different representations, as the training algorithm of RBMs is stochastic.</p>
			<p>To test this, we did a simple experiment where we trained two RBMs on the same dataset. And then we've compared the representation of this activity by looking at how correlated were the weight vectors to each hidden unit in the two models.</p>
			<p>So on the X axis here you have the hidden units of one training, and on the Y axis you have the hidden units of the other training and the colour describes their correlation with each other.</p>
			<p>If those 2 RBMs represented the activity in the same way, the diagonal would be entirely red. But we see that this is not the case. Around 35% of those hidden units have no matching pair. Then you have 40% of those hidden units where it's not very clear whether they match or not, but you still have around 30% that match perfectly. So one hidden unit in one brain will match perfectly.</p>
			<p>So this means that 2 models trained on exactly the same data can converge to 2 different representations of this activity. And so there is no way that this naive approach would work across multiple individuals.</p>
		</aside>
	</section>

	<section>
		<h1> Restricted Boltzmann Machines </h1>
		<h2> Summary </h2>
		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 3.7%; --floating-top: 12.5%; --floating-w: 1730px; --floating-h: 96px; -->
  <!--    		"> -->
		<!-- 	<p><b>Goal :</b> Find a Common Representation for the Spontaneous Brain Activity of multiple fish.</p> -->
		<!-- </div> -->

		<div class="floating fragment"
     		style="
					--floating-left: -2.0%; --floating-top: 22.1%; --floating-w: 1998px; --floating-h: 67px;
     		">
			<p style="color:red;">Spontaneous Brain Activity ⇒ no alignment</p>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: -2.1%; --floating-top: 27.6%; --floating-w: 2000px; --floating-h: 113px;
     		">
			<p style="color:red;">Neurons are not comparable across individuals</p>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: -2.1%; --floating-top: 48.5%; --floating-w: 2005px; --floating-h: 67px;
     		">
			<p style="color:green;">RBMs representation ⇒ composition of cell assemblies</p>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: -2.3%; --floating-top: 54.9%; --floating-w: 2011px; --floating-h: 73px;
     		">
			<p style="color:red;">RBMs representation ⇒ degenerate</p>
		</div>


		<div class="floating fragment"
     		style="
					--floating-left: -2.3%; --floating-top: 80.1%; --floating-w: 2008px; --floating-h: 74px;
     		">
			<p>⇒ We needs a method which <b>explicitely aligns the latent space</b> across individuals</p>
		</div>

		<aside class="notes">
			<p>Ok, let's take a quick break and summarise what we've seen so far.</p>
			<p>we're trying to find a common representation for the SA of multiple fish</p>
			<p>We've seen that because spontaneous activity is very stochastic and very different across individuals, it's very hard to compare it.</p>
			<p>And neurons are very hard to compare between individuals contrary to the brain imaging in humans</p>
			<p>but I've shown you that from whole-brain neuronal recordings, we can find a representation in terms of cell assemblies.</p>
			<p>Unfortunately, this representation is at least partially degenerate and so we cannot use it to directly compare multiple individuals.</p>
			<p>What I’ll present you now is a method that explicitly aligns this representation across multiple models trained from different individuals.</p>
		</aside>
	</section>

	<!-- <section> -->
	<!-- 	<h1> Aligning Latent Space across Individuals </h1> -->
	<!-- 	<h2> Two Methods </h2> -->
	<!---->
	<!-- 	<div class="floating" -->
 <!--     		style=" -->
	<!-- 				--floating-left: 14.6%; --floating-top: 18.5%; --floating-w: 1273px; --floating-h: 788px; -->
 <!--     		"> -->
	<!-- 			<div class="r-svg-layers"  -->
	<!-- 				data-src="assets/intro/RBM/multi_fish_best_strats.svg"  -->
	<!-- 				data-base="Layer 1"  -->
	<!-- 				data-include="Layer 1, Layer 2" -->
	<!-- 			></div> -->
	<!-- 	</div> -->
	<!---->
	<!-- </section> -->

</section> <!-- RBMs SECTION -->




<!-- <section> VOX SECTION  -->
<!---->
<!-- 	<section> -->
<!-- 		<h1> Voxelized RBM </h1> -->
<!-- 		<h2> Building a single dataset from multiple fish </h2> -->
<!---->
<!-- 		<div class="floating fragment" -->
<!--      		style=" -->
<!-- 					--floating-left: 20.2%; --floating-top: 15.0%; --floating-w: 1129px; --floating-h: 527px; -->
<!--      		"> -->
<!-- 				<div class="r-svg-layers"  -->
<!-- 					data-src="assets/multiRBM/vox_diag.svg"  -->
<!-- 					data-base="Layer 1"  -->
<!-- 				></div> -->
<!-- 		</div> -->
<!-- 		 -->
<!-- 		<div class="floating fragment" -->
<!--      		style=" -->
<!-- 					--floating-left: 5.7%; --floating-top: 62.1%; --floating-w: 1668px; --floating-h: 339px; -->
<!--      		"> -->
<!-- 				<div class="r-svg-layers"  -->
<!-- 					data-src="assets/multiRBM/vox_rasters.svg"  -->
<!-- 					data-base="Layer 1"  -->
<!-- 				></div> -->
<!-- 		</div> -->
<!---->
<!-- 	</section> -->
<!---->
<!-- 	<section> -->
<!-- 		<h1> Voxelized RBM </h1> -->
<!-- 		<h2> Hidden Space provides a Stereotypic Representation </h2> -->
<!---->
<!-- 		<div class="floating" -->
<!--      		style=" -->
<!-- 					--floating-left: 11.7%; --floating-top: 29.6%; --floating-w: 1465px; --floating-h: 472px; -->
<!--      		"> -->
<!-- 				<div class="r-svg-layers"  -->
<!-- 					data-src="assets/multiRBM/vox_shared_structure.svg"  -->
<!-- 					data-base="Layer 1"  -->
<!-- 				></div> -->
<!-- 		</div> -->
<!---->
<!-- 	</section> -->
<!---->
<!-- </section>  -->




<section> <!-- LaRBM SECTION -->
	<section>
		<h1> Aligning Representation Across Individuals </h1>
		<!-- <h2> Stereotypy of representation </h2> -->


		<div class="floating fragment"
     		style="
					--floating-left: -2.0%; --floating-top: 10.9%; --floating-w: 1998px; --floating-h: 236px;
     		">
			<p><b>Hypothesis :</b></p>
			<p>⠀</p>
			<p>⠀</p>
			<p>cell assemblies are shared across individuals</p>
			<p>⠀</p>
			<p>⠀</p>
			<p>they have the same location across individuals</p>
			<!-- <p>⇒ over the whole-brain at neuron-scale</p> -->
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: -2.6%; --floating-top: 32.5%; --floating-w: 1364px; --floating-h: 720px;
     		">
			<!-- <img src="assets/intro/RBM/multi_fish_best_strats.svg" width="100%"/> -->
				<div class="r-svg-layers" 
					data-src="assets/intro/RBM/multi_fish_best_strats.svg" 
					data-base="Layer 3" 
				></div>
		</div>

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 63.9%; --floating-top: 49.4%; --floating-w: 639px; --floating-h: 282px; -->
  <!--    		"> -->
		<!-- 	<p>Requirements</p> -->
		<!-- 	<p>⠀</p> -->
		<!-- 	<ul> -->
		<!-- 		<li>Same spatial weight distribution</li> -->
		<!-- 		<li>Same hidden space</li> -->
		<!-- </div> -->

		<aside class="notes">
			<p>The hypothesis on which this rests is that you can find the same cell assemblies in all the individuals and that they are located in the same place in the brain.</p>
			<p>And so what we are going to do is train one model per fish and then constrain those models using this spatial prior such that they have the same hidden unit representation.</p>
		</aside>
	</section>
	
	<section>
		<h1> Aligning Representation Across Individuals </h1>
		<h2> Method : Latent-aligned RBMs</h2>

		<div class="floating"
     		style="
					--floating-left: 12.2%; --floating-top: 20.0%; --floating-w: 1395px; --floating-h: 717px;
     		">
			<!-- <img src="assets/multiRBM/TS_diag.svg" width="100%"/> -->
				<div class="r-svg-layers" 
					data-src="assets/multiRBM/TS_diag.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<aside class="notes">
			<p>So the method goes like this. We start by training a first RBM on 1 fish like Thijs and Jerome have done before. We call this RBM a teacher, and we will use it to constrain a student RBMs trained on a second fish.</p>
			<p>The first step is to initialise this student.</p>
			<p>We know that each hidden unit maps to certain brain regions with certain weights, and so we can map this spatial distribution into the student fish, and use it to reconstruct, in the student, the weights to that particular hidden unit, which means that it now represents the same neuronal population in both fish.</p>
			<p>And we do this for every hidden unit to reconstruct a full weight matrix. And so with this initialization method, we have built a student RBM, which should have the same representation space as the teacher.</p>
			<p>However, at this point the student is not at all specialized to the second fish, which is its own individual and therefore needs to be trained.</p>
		</aside>
	</section>

	<section>
		<!-- <h1> Multi-fish Neuronal Model </h1> -->
		<h1> Aligning Representation Across Individuals </h1>
		<h2> Method : Latent-aligned RBMs</h2>

		<div class="floating"
     		style="
					--floating-left: 86.2%; --floating-top: -1.4%; --floating-w: 236px; --floating-h: 378px;
     		">
				<p style="font-size: 0.7em;">Jorge Fernández</p>
			  <p style="font-size: 0.7em;">⠀</p>
				<p style="font-size: 0.7em;">de Cossío Díaz</p>
			<img src="https://www.ipht.fr/wp-content/uploads/2024/09/Jorge.jpeg" width="100%"/>
		</div>

		<div class="floating"
     		style="
					--floating-left: 9.5%; --floating-top: 14.6%; --floating-w: 1491px; --floating-h: 733px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiRBM/La_LLH.svg" 
					data-base="Layer 1" 
				></div>
		</div>
		
		<div class="floating fragment"
     		style="
					--floating-left: 33.4%; --floating-top: 81.0%; --floating-w: 567px; --floating-h: 122px;
     		">
     	<ul>
			<li>Converge faster (×10)</li>
			<li>Converge more reliably</li>
     	</ul>
		</div>

		<aside class="notes">
			<p>Normally, we would train it by maximazing the likelihood of the neuronal data from the second fish.</p>
			<p>But if we do this it will slowly drift away from the teacher representation which we don't want. </p>
			<p>With Jorge we developped a method where we also constrain the student from the teacher model by generating hidden configuration from the teacher and training the student RBM to maximise the probability of those configuration.</p>
			<p>And we mix those two training objectives into one so the student learns to reproduce both the student neuronal configurations and the teacher hidden cofigurations.</p>
			<p>What we found is that this training paradigm works extremely well. We can train the student RBMs about 10 times faster than we would the teacher RBMs and they converge way more reliably, meaning that they almost systematically provide good solutions to the problem.</p>
			<p>Now we need to check two things to make sure that those 2 model do provide the same representation of brain activity.</p>
		</aside>
	</section>
	
	<section>
		<!-- <h1> Multi-fish Neuronal Model </h1> -->
		<h1> Aligning Representation Across Individuals </h1>
		<h2> Hidden Units map to Stereotypic Neuronal Populations </h2>

		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -0.0%; --floating-top: 32.4%; --floating-w: 1130px; --floating-h: 463px; -->
  <!--    		"> -->
		<!-- 		<div class="r-svg-layers"  -->
		<!-- 			data-src="assets/multiRBM/TS_maps.svg"  -->
		<!-- 			data-base="Layer 1"  -->
		<!-- 		></div> -->
		<!-- </div> -->

		<div class="floating fragment"
     		style="
					--floating-left: 12.7%; --floating-top: 12.1%; --floating-w: 1443px; --floating-h: 951px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiRBM/TS_samplemaps.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<aside class="notes">
			<p>First, we need to look at the weight distributions to check that the cell assemblies in the student do indeed correspond to the cell assemblies in the teacher.</p>
			<p>Here I'm showing you a bunch of example hidden units where on the left you have the weight distribution in the teacher fish and on the right for the student fish.</p>
			<p>So for example, at the top here you can see that this hidden unit in the teacher and in the student still represents the same functional ensembles, which most probably corresponds to pre-motor population controlling the movement of the tail to the left.</p>
			<p>And here this population is probably involved in visual processing.</p>
			<p>So overall, and even after training, the student hidden units still map to the same functional ensembles.</p>
			<p></p>
		</aside>
	</section>

	<section>
		<!-- <h1> Multi-fish Neuronal Model </h1> -->
		<h1> Aligning Representation Across Individuals </h1>
		<h2> Hidden Units describe a Shared Space </h2>

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 6.0%; --floating-top: 22.7%; --floating-w: 666px; --floating-h: 623px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/multiRBM/TS_Ph.svg" width="100%"/> -->
		<!-- </div> -->

		<!-- <div class="floating debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -4.1%; --floating-top: 22.9%; --floating-w: 610px; --floating-h: 481px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/multiRBM/QQ_Ph.svg" width="100%"/> -->
		<!-- </div> -->

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 48.6%; --floating-top: 18.6%; --floating-w: 896px; --floating-h: 743px; -->
  <!--    		"> -->
		<!-- 		<div class="r-svg-layers"  -->
		<!-- 			data-src="assets/multiDYN/HU_space_occupation.svg"  -->
		<!-- 			data-base="Layer 1"  -->
		<!-- 		></div> -->
		<!-- </div> -->
		<div class="floating fragment"
     		style="
					--floating-left: 23.1%; --floating-top: 6.1%; --floating-w: 994px; --floating-h: 966px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiRBM/SUPP_2dtrajs_fig_pres.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<aside class="notes">
			<p>The second thing we need to check is that the hidden space is the same between the teacher and the student, as we requested with our new training paradigm.</p>
			<p>Here I'm showing you the marginal distribution of an example hidden units in the teacher, and in the student. As you can see the two match perfectly.</p>
			<p>This is true for all hidden units, but I'll just show you one other example, and their joint distribution.</p>
			<p>As you can see there are regions that are extremely occupied like this one or this one. And also regions that are never explored.</p>
			<p>This means that both the teacher and student models map activity from the first and the second fish in the same sub-regions of this hidden space.</p>
			<p></p>
		</aside>
	</section>

	<section>
		<!-- <h1> Multi-fish Neuronal Model </h1> -->
		<h1> Aligning Representation Across Individuals </h1>
		<h2> Translating Activity from one Fish to Another </h2>

		<div class="floating"
     		style="
					--floating-left: 62.7%; --floating-top: -0.0%; --floating-w: 625px; --floating-h: 263px;
     		">
     	<ul>
			<li>Shared representation space</li>
			<li>Generative Model</li>
     	</ul>
			<p>⠀</p>
			<p>⇒ Same encoding ?</p>
			<p>⠀</p>
			<p>⇒ Translation</p>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 15.2%; --floating-top: 24.0%; --floating-w: 1346px; --floating-h: 797px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiRBM/deepfake_diag.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 60.2%; --floating-top: 37.1%; --floating-w: 715px; --floating-h: 409px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/multiRBM/deepfake_eq.svg" width="100%"/> -->
		<!-- </div> -->

  	<!-- <div class="entry">bidirectional generative model + same latent space → deepfake</div> -->
  	<!-- <div class="mainmessage">method → we can translate activity from on fish to the other</div> -->
  	<!-- <div class="outlook">we can now ask question : "how probable is the activity in the recipient?"</div> -->

		<aside class="notes">
			<p>Now because this space is shared between those two individuals and because we can generate data from those models, we can start to ask interesting questions like "do both fish encode brain activity the same way". To test this we can try to translate activity from one individual to the other.</p>
			<p>If we take a neuronal configuration from the teacher, we can get the representation in hidden space using the conditional probability P(h|v) as I showed earlier, transfer this representation to the teacher and reconstruct it as a neuronal configuration in the student using P(v|h). For comparison we’ll also reconstruct it in the teacher’s neuronal space.</p>
			<p>By doing this translation, we can take activity that was recorded in the teacher and try to reproduce what it would have looked like if it had also been recorded in the student. </p>
		</aside>
	</section>

	<section>
		<!-- <h1> Multi-fish Neuronal Model </h1> -->
		<h1> Aligning Representation Across Individuals </h1>
		<h2> Translating Activity from one Fish to Another </h2>
		<div class="floating"
     		style="
					--floating-left: 5.6%; --floating-top: 11.5%; --floating-w: 1677px; --floating-h: 962px;
     		">
			<video loop="true" src="assets/multiRBM/Movie3.mp4"></video>
		</div>

		<aside class="notes">
			<p>So here I'm showing you on the left the activity that we recorded in the teacher in the middle of the activity that was reconstructed in the teacher, and on the right I'm showing you activity that was translated in the student.</p>
			<p>So the two brains on the left are the teacher's brain, the brain on the right is the student brain. And the activity that we translated in the student has never really existed in the student, it's completly fake.</p>
			<p>But as you can see, it still captures most of the large scale structure of the activity in the teacher. And we like to say that this is the first ever deep fake of a zebrafish brain.</p>
		</aside>
	</section>


	<!-- <section> -->
	<!-- 	<h1> Latent-aligned RBMs </h1> -->
	<!-- 	<h2> Translating Activity from one Fish to Another </h2> -->
	<!---->
	<!-- 	<div class="floating" -->
 <!--     		style=" -->
	<!-- 				--floating-left: 48.7%; --floating-top: 3.2%; --floating-w: 927px; --floating-h: 176px; -->
 <!--     		"> -->
	<!-- 		<p>How <i>credible</i> is the translated activity?</p> -->
	<!-- 	</div> -->
	<!---->
	<!-- 	<div class="floating fragment" -->
 <!--     		style=" -->
	<!-- 				--floating-left: 8.2%; --floating-top: 19.1%; --floating-w: 1583px; --floating-h: 664px; -->
 <!--     		"> -->
	<!-- 		<img src="assets/multiRBM/deepfake_stats.svg" width="100%"/> -->
	<!-- 	</div> -->
	<!---->
	<!-- 	<div class="floating fragment debug" -->
 <!--     		style=" -->
	<!-- 				--floating-left: 47.8%; --floating-top: 80.9%; --floating-w: 950px; --floating-h: 64px; -->
 <!--     		"> -->
	<!-- 		<p>Translated configurations match recipient statistics</p> -->
	<!-- 	</div> -->
	<!---->
	<!-- </section> -->

	<section>
		<!-- <h1> Multi-fish Neuronal Model </h1> -->
		<h1> Aligning Representation Across Individuals </h1>
		<h2> Translated Activity is Probable under the Recipient Model </h2>

		<div class="floating fragment"
     		style="
					--floating-left: 10.1%; --floating-top: 19.5%; --floating-w: 1509px; --floating-h: 856px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiRBM/deepfake_freeenergy.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<aside class="notes">
			<p>We did a lot of different tests to evaluate how realistic this translated activity is, and I'll only present one of them.</p>
			<p>The idea is to evaluate how probable is this activity in the student model.</p>
			<p>To do this we use the probability of neuronal configuration according to the RBM P(v). However, this cannot actually be computed, but it can be estimated through what is called the free energy.</p>
			<p>Here I'm showing you the distribution of free energy for the teacher data evaluated in the teacher model. And as you can see, most data points are very probable, which makes sense as this is how we trained the model.</p>
			<p>And here is the free energy of configurations reconstructed from the teacher. As you can see they are slightly less probable, which makes sense as the RBM filters out some of the details of the activity.</p>
			<p>And now if we look at the activity that we translated from the student to the teacher we can see that it is as probable as the reconstructed configurations, and this is true of course when translating the other way around from teacher to student.</p>
			<p>This means that the representation of the brain activity is truly shared from the student to the teacher because we can translate it from one individual to the other while still maintaining a high probability in both.</p>
		</aside>
	</section>

	<section>
		<!-- <h1> Multi-fish Neuronal Model </h1> -->
		<h1> Aligning Representation Across Individuals </h1>
		<h2> Summary </h2>

		<div class="floating"
     		style="
					--floating-left: -2.1%; --floating-top: 11.9%; --floating-w: 2007px; --floating-h: 133px;
     		">
     	<ul>
				<li>statistical model of whole-brain spontaneous neuronal activity → RBMs</li>
				<li>RBMs representation = composition of cell assemblies</li>
     	</ul>
		</div>

		<div class="floating"
     		style="
					--floating-left: -2.9%; --floating-top: 26.8%; --floating-w: 2026px; --floating-h: 77px;
     		">
			<p>align the space of representation across multiple fish</p>
		</div>

		<div class="floating"
     		style="
					--floating-left: -2.8%; --floating-top: 38.8%; --floating-w: 2023px; --floating-h: 248px;
     		">
			<p>translate activity from one brain to another</p>
			<p>⠀</p>
			<p>⇒ same encoding across individuals</p>
			<p>⠀</p>
			<p>⇒ indistinguishable representation</p>
		</div>

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -2.1%; --floating-top: 59.5%; --floating-w: 2006px; --floating-h: 107px; -->
  <!--    		"> -->
		<!-- 	<p>The functional organization of Spontaneous Activity</p> -->
		<!-- 	<p>⠀</p> -->
		<!-- 	<p>is conserved at the level of neuronal populations</p> -->
		<!-- </div> -->

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 72.0%; --floating-top: 11.1%; --floating-w: 82px; --floating-h: 299px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/bracket.svg" width="100%"/> -->
		<!-- </div> -->
		<!---->
		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 76.9%; --floating-top: 19.0%; --floating-w: 249px; --floating-h: 125px; -->
  <!--    		"> -->
		<!-- 	<p>only static</p> -->
		<!-- 	<p>⠀</p> -->
		<!-- 	<p>description</p> -->
		<!-- </div> -->

		<div class="floating fragment"
     		style="
					--floating-left: 33.4%; --floating-top: 74.1%; --floating-w: 551px; --floating-h: 83px;
     		">
			<p>What about the dynamics?</p>
		</div>

		<aside class="notes">
			<p>Now let's take a small break again and summarise what we've seen so far.</p>
			<p>I've shown you that we can build a statistical model of the whole brain of the zebra fish at the neuronal scale, and that this model represents brain activity as a composition of cell assemblies.</p>
			<p>I've shown you a new method that allows us to align this representation across individuals and to illustrate how well this alignment work, we've translated activity from one brain to the other.</p>
			<p>I think that these results demonstrate that, even tho each fish is an individuals and is therefore different from its peers, and even tho they might be expressing different things with there brain activity, they are using the same language, at least at the representation level of the hidden units. So in hidden space, these fish look identical.</p>
			<p>In the rest of this presentation we will see if the way the different fish dynamically explore this shared space is also stereotypycal</p>
		</aside>
	</section>

</section> <!-- LARBM SECTION -->




<section> <!-- DYNAMICS SECTION -->

	<section>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> an Analogy with Behavior </h2> 

		<div class="floating fragment"
     		style="
					--floating-left: 21.7%; --floating-top: 10.7%; --floating-w: 1120px; --floating-h: 937px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/intro/behavior/Mueller2019.svg" 
					data-base="Layer 1" 
				></div>
		</div>

		<ul class="citations" style="bottom: 100px">
  		<li><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007105" target="_blank" rel="noopener">
      		<span class="author">Mueller & Ravbar et al.</span> <span class="journal">PLOS Comp. Bio.</span> <span class="year">2019</span>
    	</a></li>
		</ul>

		<aside class="notes">
			<p>To look at temporal structure, a classical method is to classify this activity into states which represent reocurring activity patterns. To make an analogy, this is typiccaly what we do to study behavior. And I'll illustrate this with my favorite example.</p>
			<p>...</p>
		</aside>
	</section>

	<section>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> Segmenting the hidden space into a vocabulary of brain states </h2>

		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -1.2%; --floating-top: 31.1%; --floating-w: 1034px; --floating-h: 488px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/intro/behavior/fly_smal.svg" width="100%"/> -->
		<!-- </div> -->

		<div class="floating fragment"
     		style="
					--floating-left: 19.7%; --floating-top: 17.4%; --floating-w: 1063px; --floating-h: 898px;
     		">
				<p>Shared space of Representation</p>
				<p>⠀</p>
				<div class="r-svg-layers" 
					data-src="assets/multiDYN/HU_space_occupation.svg" 
					data-base="Layer 1"
					data-include="Layer 1, Layer 2, Layer 4, Layer 5"
				></div>
		</div>

		<aside class="notes">
			<p>If we come back to our SA. We have built a shared space, and here again I'm only showing you a 2D slice of it for simplicity.</p>
			<p>The idea is gonna be to segment this space into states, and compare the temporal organization with which each fish moves between those states.</p>
		</aside>
	</section>

	<section>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> Segmenting the hidden space into a vocabulary of brain states </h2>
		
		<div class="floating"
     		style="
					--floating-left: 16.3%; --floating-top: 21.2%; --floating-w: 1247px; --floating-h: 736px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiDYN/sRBM_diag_complex.svg" 
					data-base="Layer 1"
				></div>
		</div>

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 65.3%; --floating-top: 18.2%; --floating-w: 557px; --floating-h: 763px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/multiDYN/sRBM_train.svg" width="100%"/> -->
		<!-- </div> -->

		<aside class="notes">
			<p>To do this we add on top of the RBM another RBM which captures the regularities of hidden units activations with a binary layer.</p>
			<p>The trick here is to realise that this vector of binary units can be understood as just a decimal number, which for us will describe our states.</p>
			<p>Because those 2 layers are the same, we can take a neuronal configuration v, get its representation in hidden space and then convert this into a binary vector which can then be translated into a state.</p>
			<p>Now before I show you what those states look like, I'm just going to tell you quickly how we train this model. The idea is that because the hidden space is shared by all individuals, we can just connect all of our different brain RBMs to this state RBMs, generate many hidden configurations from all of those brain RBMs and feed them as if it was one big dataset to the state RBM. </p>
		</aside>
	</section>
	
	<section>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> Decoding State Sequences from Neuronal Data </h2>

		<div class="floating"
     		style="
					--floating-left: 0.2%; --floating-top: 11.2%; --floating-w: 839px; --floating-h: 421px;
     		">
			<img src="assets/multiDYN/sRBM_diag.svg" width="100%"/>
		</div>

		<div class="floating fragment" data-fragment-index="1"
     		style="
					--floating-left: 54.1%; --floating-top: 22.1%; --floating-w: 748px; --floating-h: 179px;
     		">
			$$P(s\mid\mathbf{v}_t) = \mathbb{E}\Big[ P\big( s\mid P(\mathbf{h}\mid\mathbf{v}_t) \big) \Big]$$
		</div>

		<div class="floating fragment" data-fragment-index="2"
     		style="
					--floating-left: 2.6%; --floating-top: 47.6%; --floating-w: 1817px; --floating-h: 366px;
     		">
			<img src="assets/multiDYN/sRBM_Psv.svg" width="100%"/>
		</div>

		<div class="floating fragment" data-fragment-index="2"
     		style="
					--floating-left: 72.9%; --floating-top: -0.1%; --floating-w: 349px; --floating-h: 66px;
     		">
			<p>model : 32 states</p>
		</div>


		<!-- <div class="floating debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 73.9%; --floating-top: 2.9%; --floating-w: 630px; --floating-h: 501px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/multiDYN/Psv_vs_S.svg" width="100%"/> -->
		<!-- </div> -->

		<!-- <div class="floating debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -3.1%; --floating-top: 14.5%; --floating-w: 1391px; --floating-h: 293px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/multiDYN/smax.svg" width="100%"/> -->
		<!-- </div> -->

  	<!-- <div class="entry">building sequence of states</div> -->
  	<!-- <div class="mainmessage">state sequence + smax + "certainty" vs S</div> -->
  	<!-- <div class="outlook">video of state sequence</div> -->

		<aside class="notes">
			<p>Now, what do those state sequences look like? Because it's a probabilistic model, we can compute the probability of having a state s given a neuronal configuration v, which essentially means that we are sampling successively the brain RBM and then the state RBM.</p>
			<p>The sequences look like this. So in the X axis you have time, and on the Y axis you have the states, in this case 32 states. And so every column of this graph represents one time point with the probability of each state given by the colour. And as you can see, most of the time there is only one state which is probable, which means that even though this model is probabilistic, it provides a pretty much deterministic segmentation of the brain activity.</p>
		</aside>
	</section>

	<section>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> States Capture Large Scale Feature of Brain Activity </h2>

		<div class="floating"
     		style="
					--floating-left: 16.9%; --floating-top: 11.9%; --floating-w: 1227px; --floating-h: 949px;
     		">
			<video loop="true" src="assets/multiDYN/Movie1.mp4"></video>
		</div>

		<div class="floating"
     		style="
					--floating-left: 72.9%; --floating-top: -0.1%; --floating-w: 349px; --floating-h: 66px;
     		">
			<p>model : 32 states</p>
		</div>

		<aside class="notes">
			<p>Now this is what these states sequences look like in neuronal space.</p>
			<p>So on the left again you have the activity recorded in one fish, in the middle you have the activity reconstructed from the brain RBM, and on the right you have the neural representation of the most probable state, with the actual sequence at the bottom. </p>
			<p>What's very important to understand here is that we are doing a massive dimensionality reduction. On the left we are describing the activity of 40,000 neurons with in the middle only a hundred hidden units and on the right only 32 states.</p>
			<p>But even though we are doing this massive dimensionality reduction, the states still capture the large structures we observe in the data.</p>
		</aside>
	</section>

	<section>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> Across Individuals ? </h2>

		<div class="floating"
     		style="
					--floating-left: 28.2%; --floating-top: 36.9%; --floating-w: 818px; --floating-h: 199px;
     		">
     	<ul>
				<li>Are states stereoypical ?</li>
				<li>Are states used similarly ?</li>
				<li>Are states sequences stereotypical ?</li>
     	</ul>
		</div>

		<aside class="notes">
			<p>Okay, now that we have those states, we can ask three important questions. First : are those states the same in all the fish. Then : are those states used similarly by the different fish? And lastly : is the structure of the sequence the same in all the fish? So first let's look at the states themselves.</p>
		</aside>
	</section>

	<section>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> States represent Stereotyped Wrain-wide Configurations </h2>

		<div class="floating"
     		style="
					--floating-left: 19.9%; --floating-top: 19.2%; --floating-w: 1190px; --floating-h: 813px;
     		">
			<img src="assets/multiDYN/state_maps.svg" width="100%"/>
		</div>

		<div class="floating"
     		style="
					--floating-left: 72.9%; --floating-top: -0.1%; --floating-w: 349px; --floating-h: 66px;
     		">
			<p>model : 32 states</p>
		</div>

		<aside class="notes">
			<p>Here I'm showing you the neural representation of seven example states in both a teacher and a student.</p>
			<p>And as you can see, those maps are quite similar between all the individuals. There are a few differences between those maps which are expected from the fact that they are different individuals and also from the fact that we are going to accumulate noise in the model, but still we captured a large organisation which is the same across all fish.</p>
		</aside>
	</section>
	
	<section>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> States are Used Similarly by All Fish </h2>

		<div class="floating"
     		style="
					--floating-left: 72.9%; --floating-top: -0.1%; --floating-w: 349px; --floating-h: 66px;
     		">
			<p>model : 32 states</p>
		</div>

		<div class="fragment">
		<div class="floating"
     		style="
					--floating-left: 12.8%; --floating-top: 13.5%; --floating-w: 557px; --floating-h: 221px;
     		">
			$$P(s) = \frac{1}{T}\sum_{t=1}^T P(s\mid\mathbf{v}_t)$$
		</div>

		<div class="floating"
     		style="
					--floating-left: 13.5%; --floating-top: 34.3%; --floating-w: 535px; --floating-h: 510px;
     		">
			<img src="assets/multiDYN/Ps_id.svg" width="100%"/>
		</div>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 49.8%; --floating-top: 24.8%; --floating-w: 853px; --floating-h: 667px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiDYN/Ps_matrix.svg" 
					data-base="Layer 1"
				></div>
		</div>

		<!-- <div class="floating debug" -->
  <!--    		style=" -->
		<!-- 			--floating-left: 68.9%; --floating-top: 2.3%; --floating-w: 473px; --floating-h: 535px; -->
  <!--    		"> -->
		<!-- 	<img src="assets/multiDYN/PS_vs_S.svg" width="100%"/> -->
		<!-- </div> -->


  	<!-- <div class="entry">what states are the most probable</div> -->
  	<!-- <div class="mainmessage">occupation of states + clutches + vs S</div> -->
  	<!-- <div class="outlook">??</div> -->

		<aside class="notes">
			<p>The second thing that we wanted to look at is how are those states used? </p>
			<p>To do this we can look at the probability of each state P(s). Here I'm showing you the results in 2 example fish, with 1 fish on the X-axis and another on the Y-axis.</p>
			<p>And as you can see, if a state is very probable in one fish, it's also very probable in the other.</p>
			<p>If we look at this in a bit more detail by looking at all the different pairs of fish, we can see that some fish use those states more stereotypicaly than others.</p>
			<p>You have this group of 3 fish here, and this group of 2 or three fish here. What's very interesting is that 2 groups have different parents and were imaged on different weeks.</p>
			<p>While we don't have the statistical power to know whether this is statistically significant, this hints at the fact that maybe this type of model could be used to do some phenotypy in order to compare, for example, sick fish with healthy fish.</p>
		</aside>
	</section>

	<section>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> Markovian Transition Rates are partially Conserved across fish </h2>

		<div class="floating"
     		style="
					--floating-left: 72.9%; --floating-top: -0.1%; --floating-w: 349px; --floating-h: 66px;
     		">
			<p>model : 32 states</p>
		</div>

		<div class="floating"
     		style="
					--floating-left: 47.1%; --floating-top: 21.2%; --floating-w: 942px; --floating-h: 232px;
     		">
			$$P(s\to s') = \frac{ \sum_{t=1}^{T-1} P(s\mid\mathbf{v}_t) \cdot P(s'\mid\mathbf{v}_{t+1}) }{ \sum_{t=1}^{T-1} P(s\mid\mathbf{v}_t) }$$
		</div>

		<div class="floating"
     		style="
					--floating-left: 16.2%; --floating-top: 15.0%; --floating-w: 392px; --floating-h: 387px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiDYN/markov_diag.svg" 
					data-base="Layer 1, Layer 2"
				></div>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 3.7%; --floating-top: 54.0%; --floating-w: 941px; --floating-h: 464px;
     		">
			<img src="assets/multiDYN/Pss_log.svg" width="100%"/>
		</div>

		<div class="floating fragment"
     		style="
					--floating-left: 64.1%; --floating-top: 54.0%; --floating-w: 473px; --floating-h: 462px;
     		">
			<img src="assets/multiDYN/Pss_id.svg" width="100%"/>
		</div>

  	<!-- <div class="entry">what is the structure of sequence ?</div> -->
  	<!-- <div class="mainmessage">markov + transition matrix + stereotypy</div> -->
  	<!-- <div class="outlook">??</div> -->

		<aside class="notes">
			<p>Last thing that we want to look at is the structure of these state sequences, and we are going to do it like with the flies example, by looking at the markovian probabilities of transitioning from one state to the other.</p>
			<p>Here I'm showing you the transition probability matrix between of two different fish, and here an identity plot of those transitions.</p>
			<p>We find that as long as transitions were observed in both fish, they have similar probability. This big blob corresponds to transitions observed in both fish, and these are transitions observed only in one or the other.</p>
		</aside>
	</section>

	<section data-auto-animate>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> Markovian Dynamics - sub-structure and hub-states </h2>

		<div class="floating"
     		style="
					--floating-left: 72.9%; --floating-top: -0.1%; --floating-w: 349px; --floating-h: 66px;
     		">
			<p>model : 8 states</p>
		</div>

		<div class="floating fragment" data-id="famT"
     		style="
					--floating-left: 20.0%; --floating-top: 6.2%; --floating-w: 1800px; --floating-h: 1015px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiDYN/family_T_field.svg" 
					data-base="Layer 1"
					data-include="Layer 1, Layer 2, Layer 3, Layer 4, Layer 5, Layer 6, Layer 7"
				></div>
		</div>

		<aside class="notes">
			<p>We wanted to look at this transition structure in a bit more detail, and we have observed quite consistently that around 6 or 7 states tend to dominate the others. So here I showing you all the states from a model with only 8 states.</p>
			<p>Each dot is one state, the colour of the dot is how probable it is. These are the neuronal representations of each state, and these are the transitions represented as arrows with the corresponding matrix here.</p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
		</aside>
	</section>
	 
	<section data-auto-animate>
		<h1> Dynamics of Spontaneous Activity </h1>
		<h2> Markovian Dynamics - sub-structure and hub-states </h2>

		<div class="floating"
     		style="
					--floating-left: 72.9%; --floating-top: -0.1%; --floating-w: 349px; --floating-h: 66px;
     		">
			<p>model : 8 states</p>
		</div>

		<div class="floating" data-id="famT"
     		style="
					--floating-left: 1.0%; --floating-top: 6.2%; --floating-w: 1800px; --floating-h: 1015px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/multiDYN/family_T_field.svg" 
					data-base="Layer 1, Layer 2, Layer 3, Layer 4, Layer 5, Layer 6, Layer 7, Layer 8"
				></div>
		</div>
	</section>

		<aside class="notes">
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
		</aside>
</section> <!-- DYNAMICS SECTION -->



<section> <!-- CONCLUSION + END SECTION -->
	<section>
		<h1> Conclusion</h1>

		<div class="floating fragment"
     		style="
					--floating-left: 0.4%; --floating-top: 3.1%; --floating-w: 1844px; --floating-h: 1008px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/conclusion.svg" 
					data-base="Layer 1"
				></div>
		</div>

		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -2.1%; --floating-top: 13.3%; --floating-w: 2001px; --floating-h: 67px; -->
  <!--    		"> -->
		<!-- 	<p><b>New Method ⇒ </b>shared representation of SA across individuals</p> -->
		<!-- </div> -->
		<!---->
		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -2.5%; --floating-top: 35.4%; --floating-w: 2008px; --floating-h: 73px; -->
  <!--    		"> -->
		<!-- 	<p><b>Translation ⇒ </b>shared encoding of brain configurations</p> -->
		<!-- </div> -->
		<!---->
		<!-- <div class="floating fragment" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -2.2%; --floating-top: 58.2%; --floating-w: 2005px; --floating-h: 74px; -->
  <!--    		"> -->
		<!-- 	<p><b>Shared States ⇒ </b>compared dynamics of SA</p> -->
		<!-- </div> -->

		<aside class="notes">
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
		</aside>
	</section>

	<section>
		<h1>Perspectives</h1>

		<div class="floating fragment"
     		style="
					--floating-left: 0.4%; --floating-top: 3.1%; --floating-w: 1844px; --floating-h: 1008px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/perspectives.svg" 
					data-base="Layer 1"
				></div>
		</div>

		<!-- Methods -->
		<!-- Teacher/Student → parallel training -->
		<!-- Segmentation + Markov Chain → HMM -->
		<!---->
		<!-- Applications -->
		<!-- Predicting :  -->
		<!-- 	- evoked responses across individuals -->
		<!-- 	- responses to optogenetic perturbations -->
		<!-- Phenotyping :  -->
		<!-- 	- mutants -->
		<!-- 	- pharmacological screenings -->
		<!-- Multi-session / Multi-individuals :  -->
		<!-- 	- learning and memory -->
		<!-- 	- development -->
		<!-- 	- increased statistical power -->
		<!---->
		<!-- Question : -->
		<!-- 	Functional interpretability of states and transitions -->

		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -2.4%; --floating-top: 29.8%; --floating-w: 2010px; --floating-h: 280px; -->
  <!--    		"> -->
  <!--    	<ul> -->
				<!-- <li>Spontaneous Activity is challenging to compare between individuals</li> -->
		<!-- 		<li>RBMs provide a representation of whole brain activity as a composition of cell assemblies</li> -->
		<!-- 		<li>Method : align this representation across multiple zebrafish larvae</li> -->
		<!-- 		<li>Method : define shared brain states and compare their temporal organization</li> -->
		<!-- 		<li>Spontaneous Brain Dynamics is partially conserved across individuals</li> -->
  <!--    	</ul> -->
		<!-- </div> -->
		<!---->
		<!-- <div class="floating" -->
  <!--    		style=" -->
		<!-- 			--floating-left: -1.9%; --floating-top: 75.5%; --floating-w: 2002px; --floating-h: 191px; -->
  <!--    		"> -->
  <!--    	<ul> -->
		<!-- 		<li>Joint training of RBMs (instead of Teacher/Student)</li> -->
		<!-- 		<li>Integrate a temporal model directly into the RBM (eg. HMM)</li> -->
		<!-- 		<li>Joint neuronal and behavioral studies to improve model interpretation</li> -->
  <!--    	</ul> -->
		<!-- </div> -->

		<aside class="notes">
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
		</aside>
	</section>

	<section>
		<h1> Joint Behavioral and Neuronal States </h1>
		<h2> Zebrafish Reorientation </h2>

		<div class="floating"
     		style="
					--floating-left: 4.0%; --floating-top: 11.7%; --floating-w: 1712px; --floating-h: 891px;
     		">
				<div class="r-svg-layers" 
					data-src="assets/hmm.svg" 
					data-base="Layer 1"
				></div>
		</div>

		<ul class="citations" style="bottom: 100px">
  		<li><a href="https://www.biorxiv.org/content/10.1101/2024.11.22.624881v1" target="_blank" rel="noopener">
      		<span class="author">Dommanget-Kott & Fernandez-de-Cossio-Diaz, Coraggioso, Bormuth, Monasson, Debrégeas, and Cocco</span> <span class="journal">bioRxiv</span> <span class="year">2024</span>
    	</a></li>
		</ul>

		<aside class="notes">
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
		</aside>
	</section>

	<section>
		<h1> Ethics </h1>
		<div class="floating"
     		style="
					--floating-left: 1.1%; --floating-top: -5.4%; --floating-w: 1859px; --floating-h: 1244px;
     		">
			<video loop="true" style="transform: rotate(90deg)" src="assets/intro/zebra/lots_of_little_fish.mp4" width="100%"></video>
		</div>

		<aside class="notes">
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
		</aside>
	</section>

	<!-- <section> -->
	<!-- 	<h1> SciComm </h1> -->
 <!--  	<div class="entry"></div> -->
 <!--  	<div class="mainmessage"></div> -->
 <!--  	<div class="outlook"></div> -->
	<!-- </section> -->

	<section>
		<div class="floating"
     		style="
					--floating-left: 1.8%; --floating-top: -0.1%; --floating-w: 1865px; --floating-h: 1070px;
     		">
		<img src="assets/thanks.svg" width="100%"/>
		</div>
	</section>

		<aside class="notes">
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
			<p></p>
		</aside>
</section> <!-- CONCLUSION + END SECTION -->



<section> <!-- SUPPS SECTION -->
	<section>
		<h1> Supplementary </h1>
	</section>
</section> <!-- SUPPS SECTION -->












	<!-- <section> -->
	<!-- 	<video loop="true" src="https://zenodo.org/records/16886749/files/Movie1.mp4"></video> -->
	<!-- </section> -->



<section>

	<section>
		<h1>Section Title</h1>
		<h2>Smaller subtitle 1</h2>
		$$\mathcal{U}(x) = \frac{1}{2}\gamma_+ x_+^2 + \frac{1}{2}\gamma_- x_-^2 + \theta_+x_+ + \theta_-x_-$$
		<ul class="citations">
  		<li><a href="https://doi.org/xx.xxxx/xxxxx" target="_blank" rel="noopener">
      		<span class="author">Smith, J.</span> <span class="journal">Nature</span> <span class="year">2021</span>
    	</a></li>
		</ul>
	</section>



	<section>
		<h1>Section Title</h1>
		<h2>Smaller subtitle 2</h2>
		<div class="r-svg-layers" 
			data-src="assets/test.svg" 
			data-base="layer1" 
		></div>
		hey hey heey
  	<div class="entry">introduce contrastive loss → intuition first</div>
  	<div class="mainmessage">The model fails when negatives are too easy</div>
  	<div class="outlook">segue: hardness-aware sampling</div>
	</section>
	

	<section>
		<h1>Section Title</h1>
		<h2>Smaller subtitle 3</h2>

		<div class="floating debug"
     		style="
     			--floating-left: 69.7%; --floating-top: 4.8%; --floating-w: 596px; --floating-h: 204px;
     		">
  		<p>Anything goes here: text, images, charts…</p>
		</div>

		<div class="floating"
     		style="
     			--floating-left: 20.7%; --floating-top: 3.4%; --floating-w: 1071px; --floating-h: 948px;
     		">
			<img src="assets/test.svg" />
		</div>

	</section>

</section>


			</div>
		</div>

		<script src="dist/reveal.js"></script>
		<script src="plugin/notes/notes.js"></script>
		<script src="plugin/reveal-svg-layers.umd.js"></script>
		<script src="plugin/zoom/zoom.js"></script>
		<script src="plugin/math/math.js"></script>
		<script src="plugin/floating-dev-helper.js"></script>
		<script>
			Reveal.initialize({
				width: 1920, height: 1080, minScale:0.2, maxScale:1.8,
				center: false,
				controls: false, progress:false,
				slideNumber: "h.v",
				hash: true,
				navigationMode: 'linear',
				autoPlayMedia: true,
				transition: 'none',
				display: "inherit",
				defaultTiming: 52, // 45min
				plugins: [ RevealNotes, SvgLayers, RevealZoom, RevealMath.KaTeX],
			});
		</script>
	</body>
</html>
